content/administration/install/install.rst

```diff
@@ -508,37 +508,27 @@ create a new PostgreSQL user:
 Dependencies
 ^^^^^^^^^^^^
 
-For libraries using native code, it is necessary to install development tools and native
-dependencies before the Python dependencies of Odoo. They are available in `-dev` or `-devel`
-packages for Python, PostgreSQL, libxml2, libxslt1, libevent, libsasl2 and libldap2.
-
-On Debian/Unbuntu, the following command should install all the required libraries:
+The preferred way of installing dependencies is by using your distribution packages.
+For Debian based systems, those packages are listed in the `debian/control` file of the odoo sources.
+On Debian/Ubuntu, the following commands should install the required packages:
 
 .. code-block:: console
 
-    $ sudo apt install python3-dev libxml2-dev libxslt1-dev libldap2-dev libsasl2-dev \
-        libtiff5-dev libjpeg8-dev libopenjp2-7-dev zlib1g-dev libfreetype6-dev \
-        liblcms2-dev libwebp-dev libharfbuzz-dev libfribidi-dev libxcb1-dev libpq-dev
-
-Odoo dependencies are listed in the `requirements.txt` file located at the root of the Odoo
-community directory.
+    $ cd /CommunityPath
+    $ sed -n -e '/^Depends:/,/^Pre/ s/ python3-\(.*\),/python3-\1/p' debian/control | sudo xargs apt-get install -y
 
 .. tip:: It can be preferable to not mix python modules packages between different instances of Odoo
          or with your system. You can use virtualenv_ to create isolated Python environments.
+         In that case, the `requirements.txt` file can be used to install the dependencies.
```

Comment:
I think that we should specify here that -dev packages are required. It is sad to loose this list.

Reply:
That means bringing back the whole pip explanation in the tip ... what do you think @AntoineVDV